src/main/java/uk/gov/companieshouse/overseasentitiesapi/controller/OverseasEntitiesDataController.java

```diff
@@ -109,4 +112,39 @@ private ResponseEntity<OverseasEntityDataApi> getOverseasEntityDataResponse(
         
         return ResponseEntity.ok(overseasEntityDataApi);
     }
+
+    @GetMapping("/beneficial-owners")
+    public ResponseEntity<PrivateBoDataListApi> getOverseasEntityBeneficialOwners(
+            @PathVariable(TRANSACTION_ID_KEY) String transactionId,
+            @PathVariable(OVERSEAS_ENTITY_ID_KEY) String overseasEntityId,
+            @RequestHeader(value = ERIC_REQUEST_ID_KEY) String requestId) throws ServiceException {
+
+        final var logMap = new HashMap<String, Object>();
+        logMap.put(OVERSEAS_ENTITY_ID_KEY, overseasEntityId);
+        logMap.put(TRANSACTION_ID_KEY, transactionId);
+        ApiLogger.infoContext(requestId, "Calling service to retrieve submitted beneficial owner information", logMap);
+
+        PrivateBoDataListApi privateBeneficialOwnersData =  null;
```

Comment:
Move this declaration to line 134 to avoid initialising to null. 
Move line 148 to be before 142 so that all code paths in "if/else" return something and no need for catch all return.